OAuth/FirebaseOAuthUI/FUIOAuth.m

```diff
@@ -118,6 +123,7 @@ - (instancetype)initWithAuthUI:(FUIAuth *)authUI
     _signInLabel = buttonLabelText;
     _shortName = shortName;
     _buttonBackgroundColor = buttonColor;
+    _buttonTextColor = [UIColor colorWithRed:1.0 green:1.0 blue:1.0 alpha:1.0];
```

Comment:
nit: Can this just be `[UIColor whiteColor]`?

Reply:
Sure! Just tried to follow the style of the current code. Will update.